internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -1347,12 +1347,12 @@ private ConfigKeys() {
   /**
    * The job tracker identifier as part of a map reduce job id.
    */
-  public static final String PUSH_JOB_MAP_REDUCE_JT_ID = "push.job.map.reduce.jt.id";
+  public static final String PUSH_JOB_COMPUTE_JOB_ID = "push.job.compute.job.id";
```

Comment:
Just a question vs. an actual issue.  I suspect this is filled in exclusively programmatically, theres no one that we break by changing this string?

Reply:
Yep. This is set and used only by the reducers